core/lexicon/en/workspace.inc.php

```diff
@@ -152,14 +152,14 @@
 $_lang['packages'] = 'Packages';
 $_lang['packages_in_category'] = 'Below is a list of the packages available in the selected Category.';
 $_lang['packages_browse_msg'] = 'Click details on a package to get more information. Click Download to download the package to your MODX Revolution install.';
-$_lang['packages_desc'] = 'Packages are collections of MODX components that can be used within MODX in a variety of ways. Here you can manage them, update their installation status, disable or enable installed packages, or download new ones from remote providers.';
+$_lang['packages_desc'] = 'Packages are collections of MODX extras that can be used within MODX in a variety of ways. Here you can manage them, update their installation status, disable or enable installed packages, or download new ones from remote providers.';
```

Comment:
This is confusing the issue. Packages are collections of MODX components that form an extra. If anything this should be changed to "Extras are packages of MODX components…" This is making my head hurt.